src/patterns/components/card/react/info/base.hbs

```diff
@@ -0,0 +1,56 @@
+<p class="sprk-u-mbm">
+  See below for available customization options:
+</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Prop</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        additionalCardClasses
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        Expects a space-separated list of class names to be
+        appended to the class attribute of SprkCard.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        idString
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        The value supplied will be assigned to the 'data-id'
+        attribute on the component. This is intended to be used
+        as a selector for automated tools. This value should be
+        unique per page.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        standout
```

Comment:
should this be `isStandout` or `hasStandout`?

Reply:
I agree on that @bobwise

Reply:
changed!